applications/tube-network/tube_network_example/README.md

```diff
@@ -1,68 +1,33 @@
 # Tube Network Example
 
-Here we demonstrate a way of modelling the London Underground Network using data acquired from the Transport for London (TFL) website. 
+Here we demonstrate a way of modelling the London Underground Network using data acquired from the Transport for London (TFL) website.
 
 Most of the code given here is written in Python to give an example of how to use the Grakn Python client.
 
 See the quickstart for how to get going immediately, or read on for more info.
 
+## Prerequisites
+- Grakn >= 1.4.2. Learn more about [installing and running Grakn](http://dev.grakn.ai/docs/running-grakn/install-and-run).
```

Comment:
1.4.3

Reply:
do you think it should still be >= 1.4.3, even if I've tested and verified that it also works with 1.4.2?